### PR TITLE
2070: Get a placeholder message spot for integration message

### PR DESCRIPTION
**Pull request by zhaosongzs**
As Magnus said in the issue description, sometimes it's hard for users to find the merge ready comment. So he would like to see that this message is added as soon as possible, after the bridgekeeper has done its job. 

In this patch, on the very beginning of CheckWorkItem, skara bot would post a placeholder message to the PR and update it later.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace

### Issue
 * [SKARA-2070](https://bugs.openjdk.org/browse/SKARA-2070): Get a placeholder message spot for integration message (**Bug** - P3)

### Reviewers
 * [Erik Joelsson](https://openjdk.org/census#erikj) (@erikj79 - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/skara.git pull/1619/head:pull/1619` \
`$ git checkout pull/1619`

Update a local copy of the PR: \
`$ git checkout pull/1619` \
`$ git pull https://git.openjdk.org/skara.git pull/1619/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1619`

View PR using the GUI difftool: \
`$ git pr show -t 1619`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/skara/pull/1619.diff">https://git.openjdk.org/skara/pull/1619.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/skara/pull/1619#issuecomment-1986576829)

### CODE REVIEW

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java [hunk 1]
```diff
@@ -721,6 +725,23 @@ private boolean unhandledIntegrateCommand(List<Comment> comments) {
...
+        var existing = comments.stream()
+                .filter(comment -> comment.author().equals(pr.repository().forge().currentUser()))
+                .filter(comment -> comment.body().contains(MERGE_READY_MARKER))
+                .findAny();
+        if (existing.isPresent()) {
+            return comments;
+        }
+        String message = "❗ This change is not yet ready to be integrated.\n" +
+                "See the **Progress** checklist in the description for automated requirements.\n" +
+                "This project also has non-automated pre-integration requirements. Please see the file CONTRIBUTING.md for details.\n" +
```
[the remark this answers is not in the document]
Comment (zhaosongzs) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java:738]: Yes, we shouldn't include that message. I will fix it. Thanks!

**Comment by zhaosongzs** [comment]
> Nice! Thanks for fixing this. Do you have an test PR that demonstrates how this looks?

No, I haven't got a test PR. I will have one on playground after this PR is integrated.

**Review by erikj79 — APPROVED** [review]

**Comment by zhaosongzs** [comment]
Thanks for the review!
/integrate

**Comment by zhaosongzs** [comment]
> Nice! Thanks for fixing this. Do you have an test PR that demonstrates how this looks?

Here is the test PR. You can see the edit history of this comment.
https://github.com/openjdk/playground/pull/185#issuecomment-1992192423

**Pull request closed without merging**